src/elements/form/Control.ts

```diff
@@ -99,6 +99,10 @@ export class NovoCustomControlContainerElement {
     selector: 'novo-control',
     template: `
         <div class="novo-control-container" [formGroup]="form" [hidden]="form.controls[control.key].hidden || form.controls[control.key].type === 'hidden' || form.controls[control.key].controlType === 'hidden'">
+            <!--Encrypted Field-->
+            <i [hidden]="!isFieldEncrypted(control)"
```

Comment:
Don't want to call a function inside of interpolation. Angular won't know how to handle change detection for this and it will slow down forms. 

How are you marking the field as encrypted? Just putting the words `customEncrypted` on the key? Why not add a new property to the BaseControl, propagate it onto the NovoFormControl and then you can just `form.controls[control.key].encrypted` here?

Reply:
Also, does the field need to render differently if it's encrypted? Should it be disabled or obfuscated like a password field?

Reply:
Good call, working on making those changes. As for the purpose, just needed to make an indicator that the data is encrypted in the db. What you are talking about is confidential fields, which we are implementing in another epic.